R/tostring.R

```diff
@@ -31,6 +31,7 @@ d_hsep_factory <- function() {
 #' if in a locale that uses a UTF character set, otherwise an ASCII hyphen
 #' with a once-per-session warning.
 #'
+#' @keyword internal
```

Comment:
Used by rtables (and, I assume without bothering ot check, rlistings) to get default horizontal separator across UTF and non-UTF locales (in fact thats its entire purpose), can't be made internal.